File: B/cpp/FootballTeam.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

class FootballTeam
{
    private:
        string name;
        string origin;
        string year;
        vector <string> players;

    // set default data
    public:
        FootballTeam(){}

        FootballTeam(string name, string origin, string year, vector <string> players)
        {
            this->name = name;
            this->origin = origin;
            this->year = year;
            this->players.insert(this->players.end(), players.begin(), players.end());
        }

        void setName(string name)
        {
            this->name = name;
        }

        string getName()
        {
            return this->name;
        }

        void setOrigin(string origin)
        {
            this->origin = origin;
        }

        string getOrigin()
        {
            return this->origin;
        }

        void setYear(string year)
        {
            this->year = year;
        }

        string getYear()
        {
            return this->year;
        }

        void setPlayers(vector <string> players)
        {
            this->players.insert(this->players.end(), players.begin(), players.end());
        }

        vector <string> getPlayers()
        {
            return this->players;
        }

        void addPlayer(string player)
        {
            this->players.push_back(player);
        }

        void showTeam()
        {
            cout << "Team Name      : " << this->getName() << endl;
            cout << "Origin Country : " << this->getOrigin() << endl;
            cout << "Year of Build  : " << this->getYear() << endl;

            for(int i = 0; i < this->getPlayers().size(); i++)
            {
                cout << "    - " << this->getPlayers()[i] << endl;
            }
        }
        ~FootballTeam(){}
};

File: B/java/FootballTeam.java
public class FootballTeam
{
    private String name;
    private String origin;
    private String year;
    private String[] players;

    // set default data
    public FootballTeam(){}

    public FootballTeam(String name, String origin, String year, String[] players)
    {
        this.name = name;
        this.origin = origin;
        this.year = year;
        this.players = players;
    }

    public void setName(String name)
    {
        this.name = name;
    }

    public String getName()
    {
        return this.name;
    }

    public void setOrigin(String origin)
    {
        this.origin = origin;
    }

    public String getOrigin()
    {
        return this.origin;
    }

    public void setYear(String year)
    {
        this.year = year;
    }

    public String getYear()
    {
        return this.year;
    }

    public void setPlayers(String[] players)
    {
        this.players = players;
    }

    public String[] getPlayers()
    {
        return this.players;
    }

    public void addPlayer(String player)
    {
        this.players[] = player;
    }

    public void showTeam()
    {
        System.out.println("Team Name      : " + this.getName());
        System.out.println("Origin Country : " + this.getOrigin());
        System.out.println("Year of Build  : " + this.getYear());

        String[] players = this.getPlayers();
        for(int i = 0; i < players.length; i++)
        {
            System.out.println("    - " + players[i]);
        }
    }
}